sql/core/src/test/resources/sql-tests/results/ansi/interval.sql.out

```diff
@@ -239,15 +239,15 @@ NULL
 -- !query 29
 select -interval '-1 month 1 day -1 second'
 -- !query 29 schema
-struct<1 months -1 days 1 seconds:interval>
+struct<(- -1 months 1 days -1 seconds):interval>
```

Comment:
We should change the default alias of interval literal in a followup.
```
scala> sql("select interval '1 day', date '2000-10-10'").show
+------+-----------------+
|1 days|DATE '2000-10-10'|
+------+-----------------+
|1 days|       2000-10-10|
+------+-----------------+
```
We should follow date and pick `INTERVAL '1 days'`